test/shaders/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+#######################
+cmake_minimum_required(VERSION 3.15)
+
+add_library(test_shaders "Utils.cpp"
```

Comment:
Why is this being set up as a separate library?

Reply:
It's discussed later on the tests area around the thinking to explore simplicity

Reply:
Logically separating stuff and reducing compile time.